docs/current_docs/quickstart/183917-daggerverse.mdx

```diff
@@ -0,0 +1,18 @@
+---
+slug: /quickstart/183917/daggerverse
+hide_table_of_contents: true
+title: "Use modules from the Daggerverse"
+---
+
+# Quickstart
+
+## Use modules from the Daggerverse
+
+As you've seen in previous sections, modules can be consumed from remote sources like GitHub, or installed and used locally. The [Daggerverse](https://daggerverse.dev) is an online catalog of Dagger modules that enables users to easily discover and use modules created by the Dagger community.
```

Comment:
```suggestion
As you've seen in previous sections, functions are designed to be easily shared and reused: all you need it the git URL of a module, and you can call all the functions from that module.

But how to find useful modules? Enter *the Daggerverse*.

The [Daggerverse](https://daggerverse.dev) is a free service run by Dagger, which indexes all publicly available Dagger modules, and lets you easily search and consume them.

Using the Daggerverse, you can easily discover great modules being developed by the community, learn how they work, and start using them. Since modules are just source code, it's easy to contribute to them, too!
```